Fix a number of compile errors like the following on armhf:

  In file included from /<<PKGBUILDDIR>>/obj-arm-linux-gnueabihf/dom/media/platforms/ffmpeg/ffmpeg60/Unified_cpp_ffmpeg_ffmpeg600.cpp:83:
  /<<PKGBUILDDIR>>/dom/media/platforms/ffmpeg/FFmpegVulkanVideoDecoder.cpp:120:21: error: assigning to 'VkImage' (aka 'unsigned long long') from incompatible type 'std::nullptr_t'
    120 |     mNv12Image[i] = nullptr;
        |                     ^~~~~~~
  /<<PKGBUILDDIR>>/dom/media/platforms/ffmpeg/FFmpegVulkanVideoDecoder.cpp:121:19: error: assigning to 'VkDeviceMemory' (aka 'unsigned long long') from incompatible type 'std::nullptr_t'
    121 |     mNv12Mem[i] = nullptr;
        |                   ^~~~~~~

--- a/dom/media/platforms/ffmpeg/FFmpegVulkanVideoDecoder.cpp
+++ b/dom/media/platforms/ffmpeg/FFmpegVulkanVideoDecoder.cpp
@@ -117,8 +117,8 @@ void FFmpegVideoDecoder<LIBAV_VER>::FFmp
   mDeviceFunctions.Clear();
 
   for (int i = 0; i < kNumBuffers; i++) {
-    mNv12Image[i] = nullptr;
-    mNv12Mem[i] = nullptr;
+    mNv12Image[i] = 0;
+    mNv12Mem[i] = 0;
     mNv12BaseFd[i] = -1;
     mCopyDoneSem[i] = VK_NULL_HANDLE;
     mCopyDoneSemFd[i] = -1;
@@ -796,11 +796,11 @@ FFmpegVideoDecoder<LIBAV_VER>::FFmpegVul
     }
     if (mNv12Image[buf]) {
       mDestroyImage(mDevice, mNv12Image[buf], nullptr);
-      mNv12Image[buf] = nullptr;
+      mNv12Image[buf] = 0;
     }
     if (mNv12Mem[buf]) {
       mFreeMemory(mDevice, mNv12Mem[buf], nullptr);
-      mNv12Mem[buf] = nullptr;
+      mNv12Mem[buf] = 0;
     }
   }
 
@@ -842,11 +842,11 @@ FFmpegVideoDecoder<LIBAV_VER>::FFmpegVul
       }
       if (mNv12Mem[b]) {
         mFreeMemory(mDevice, mNv12Mem[b], nullptr);
-        mNv12Mem[b] = nullptr;
+        mNv12Mem[b] = 0;
       }
       if (mNv12Image[b]) {
         mDestroyImage(mDevice, mNv12Image[b], nullptr);
-        mNv12Image[b] = nullptr;
+        mNv12Image[b] = 0;
       }
     }
   });
@@ -924,7 +924,7 @@ FFmpegVideoDecoder<LIBAV_VER>::FFmpegVul
     }
     if (memTypeIndex == UINT32_MAX) {
       mDestroyImage(mDevice, mNv12Image[buf], nullptr);
-      mNv12Image[buf] = nullptr;
+      mNv12Image[buf] = 0;
       return MediaResult(
           NS_ERROR_DOM_MEDIA_FATAL_ERR,
           RESULT_DETAIL("No compatible memory type for NV12 image"));
@@ -944,7 +944,7 @@ FFmpegVideoDecoder<LIBAV_VER>::FFmpegVul
     res = mAllocateMemory(mDevice, &allocInfo, nullptr, &mNv12Mem[buf]);
     if (res != VK_SUCCESS) {
       mDestroyImage(mDevice, mNv12Image[buf], nullptr);
-      mNv12Image[buf] = nullptr;
+      mNv12Image[buf] = 0;
       return MediaResult(NS_ERROR_DOM_MEDIA_FATAL_ERR,
                          RESULT_DETAIL("Failed to alloc NV12 memory"));
     }
@@ -953,8 +953,8 @@ FFmpegVideoDecoder<LIBAV_VER>::FFmpegVul
     if (res != VK_SUCCESS) {
       mFreeMemory(mDevice, mNv12Mem[buf], nullptr);
       mDestroyImage(mDevice, mNv12Image[buf], nullptr);
-      mNv12Mem[buf] = nullptr;
-      mNv12Image[buf] = nullptr;
+      mNv12Mem[buf] = 0;
+      mNv12Image[buf] = 0;
       return MediaResult(NS_ERROR_DOM_MEDIA_FATAL_ERR,
                          RESULT_DETAIL("Failed to bind NV12 memory"));
     }
@@ -981,8 +981,8 @@ FFmpegVideoDecoder<LIBAV_VER>::FFmpegVul
     if (res != VK_SUCCESS) {
       mFreeMemory(mDevice, mNv12Mem[buf], nullptr);
       mDestroyImage(mDevice, mNv12Image[buf], nullptr);
-      mNv12Mem[buf] = nullptr;
-      mNv12Image[buf] = nullptr;
+      mNv12Mem[buf] = 0;
+      mNv12Image[buf] = 0;
       return MediaResult(NS_ERROR_DOM_MEDIA_FATAL_ERR,
                          RESULT_DETAIL("Failed to export NV12 FD"));
     }
